buildenv/docker/mkdocker.sh

```diff
@@ -526,9 +524,12 @@ fi
   echo "    cpio \\"
   echo "    curl \\"
   echo "    file \\"
-if [ $arch != s390x ] ; then
-  echo "    g++-7 \\"
-  echo "    gcc-7 \\"
+if [ $version = 18.04 ] ; then
+  echo "    g++-8 \\"
+  echo "    gcc-8 \\"
+else
+  echo "    g++-10 \\"
```

Comment:
Do you want to use 11? I've used it and it has worked to compile all versions.

Reply:
Although I'm not sure if it's available on ub20, which could be why you didn't use it.

Reply:
It wasn't available on Ubuntu 20.04 when I checked.

Reply:
It is available with Ubuntu:22.04, but the script needs other changes for that.